.github/workflows/unit-tests.yml

```diff
@@ -22,17 +22,17 @@ jobs:
           cache: true
 
       - name: Run all unit tests.
-        run: make test-coverage
+        run: make test-unit
```

Comment:
In the `unit-test.yml` GitHub action, there's no reason to save a coverage.txt since it's not published or visible. We're actually using the reports from `integration-test.yml`. We can use `make test-unit` to have the CI run faster.